src/common/transformations/tests/common_optimizations/select_with_one_value_condition_test.cpp

```diff
@@ -128,3 +128,21 @@ static const std::vector<SelectWithOneValueConditionParams> params = {
 };
 
 INSTANTIATE_TEST_SUITE_P(SelectWithOneValueConditionTest, SelectWithOneValueConditionTest, ValuesIn(params));
+
+TEST(TransformationTests, SelectWithOneValueCondition_DontRenameParameter) {
```

Comment:
it's better to add an additional test when `broadcast` appears in the graph

Reply:
I just removed the changes for the branch with `broadcast` as irrelevant for this fix so it's not needed - with `broadcast` as replacement `parameter` won't be overwritten.